nilearn/plotting/img_plotting.py

```diff
@@ -721,12 +735,15 @@ def plot_glass_brain(stat_map_img,
         black_bg: boolean, optional
             If True, the background of the image is set to be black. If
             you wish to save figures with a black background, you
-            will need to pass "facecolor='k', edgecolor='k'" to pylab's
-            savefig.
+            will need to pass "facecolor='k', edgecolor='k'" to plt.savefig.
         cmap: matplotlib colormap, optional
             The colormap for specified image
         alpha: float between 0 and 1
             Alpha transparency for the brain schematics
+        vmin: float
```

Comment:
Since glass brain does not currently support negative values, do we need some additional test, not required for the other plot_*?